controller/api.go

```diff
@@ -78,6 +105,7 @@ func (api *API) Serve(port int) {
 	r.HandleFunc("/v1/functions/{function}", api.FunctionApiGet).Methods("GET")
 	r.HandleFunc("/v1/functions/{function}", api.FunctionApiUpdate).Methods("PUT")
 	r.HandleFunc("/v1/functions/{function}", api.FunctionApiDelete).Methods("DELETE")
+	r.HandleFunc("/v1/functions/{function}/logs/{dbType}", api.FunctionLogsApiPost).Methods("POST")
```

Comment:
Thinking about this path: It seems the `{function}` variable is not really being used? The query from the CLI has the function name. 

Also this isn't really part of the versioned API. The API is still influxdb's, this is just a proxy.

How about simply `/proxy/{dbType}`?